modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js

```diff
@@ -1294,7 +1294,7 @@ AUI.add(
 						var defaultLocale = instance.getDefaultLocale();
 
 						if (
-							locale === defaultLocale ||
+							!(locale in localizationMap) ||
```

Comment:
Hey @manuelRives, can you please give more details about this specific change? Why did you need to do it?

Reply:
Hi @natocesarrego, the old condition of locale === defaultLocale was used to allow the save when when te user has not modified the text because on that situation the localizationMap is an empty object. But the problem is that if the locale is different than the defaultLocale this condition is not fullfilled.

On the other hand with my new condition, if the locale is not stored on the locationMap, it also means that the user has not modified the text, since the localizationMap will have the avaliableLocales unless the the user has not modified the text. And on top, this condition works independently of the locale of the page, because on localized pages that Locale doesn't have to be the same one as the defaultLocale.

The lack of that second consideration was causing the problem that I identified on this LPS.

I hope this solve your question.

Reply:
Gotcha, @manuelRives! It makes sense to me. :slightly_smiling_face: 

Thanks for your clarification